### PR TITLE
Checking in the google-photos codelab, attempt #2

### PR DESCRIPTION
**Pull request by domesticmouse**
This time with .gitignore files =)

### CODE REVIEW

File: photos-sharing/final/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# How to Contribute
```
Comment (jfschmakeit) [inline photos-sharing/final/CONTRIBUTING.md:1]: Remove as there's already a CONTRIBUTING file at the top level.

File: photos-sharing/final/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# Google Photos Library API Sharing Codelab
+
+This branch contains the final implemented app.
+
+See the `main` branch for the initial version of the app that you should check out to begin the
```
Comment (jfschmakeit) [inline photos-sharing/final/README.md:5]: > `initial` directory

File: photos-sharing/initial/README.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+## What you’ll need
+
+* A Google Developers account and project
+* Two Google user accounts that have access to Google Photos
+* Flutter development environment set up, including two emulators or devices
+
+
+## Getting started
+
+Check out the `main` branch of this repository and follow the steps on
```
Comment (jfschmakeit) [inline photos-sharing/initial/README.md:26]: > `initial` directory

File: photos-sharing/final/LICENSE [hunk 4]
```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```
Comment (jfschmakeit) [inline photos-sharing/final/LICENSE:1]: Is a dedicated LICENSE file needed in these sub directories?

**Review by jfschmakeit — CHANGES_REQUESTED** [review]
LGTM, just need to update the READMEs (and clean up the LICENSE files if needed)

File: photos-sharing/final/LICENSE (hunk 4 above)
Comment (domesticmouse) [reply photos-sharing/final/LICENSE:1]: It’s a different license to most of the other codelabs in this repo, so I’d prefer to leave it as is. Keep OSPO happy. 😎

File: photos-sharing/initial/README.md (hunk 3 above)
Comment (domesticmouse) [reply photos-sharing/initial/README.md]: done

File: photos-sharing/final/README.md (hunk 2 above)
Comment (domesticmouse) [reply photos-sharing/final/README.md]: done

File: photos-sharing/final/CONTRIBUTING.md (hunk 1 above)
Comment (domesticmouse) [reply photos-sharing/final/CONTRIBUTING.md]: done for both `photos-sharing/initial` and `photos-sharing/final`

File: photos-sharing/final/LICENSE (hunk 4 above)
Comment (jfschmakeit) [reply photos-sharing/final/LICENSE:1]: SGTM, thanks for clarifying!

**Review by jfschmakeit — APPROVED** [review]
Thanks, I think we are all good now!

**Comment by domesticmouse** [comment]
Thanks JF!

**Pull request merged**